lib/cisco_node_utils.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
```

Comment:
Are you updating the copyright date on every file or only on the ones that have actually been touched in 2016?